src/main/java/picard/sam/ReplaceSamHeader.java

```diff
@@ -53,8 +53,8 @@
         programGroup = SamOrBam.class
 )
 public class ReplaceSamHeader extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Replaces the SAMFileHeader in a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool makes it possible to replace the header of a SAM or BAM file with the header of another" +
+    static final String USAGE_SUMMARY = "Replaces the SAMFileHeader in a SAM, BAM or CRAM file.  ";
+    static final String USAGE_DETAILS = "This tool makes it possible to replace the header of a SAM, BAM or CRAM  file with the header of another" +
             "file, or a header block that has been edited manually (in a stub SAM file). The sort order (@SO) of the two input files must " +
             "be the same.<br /><br />" +
             "Note that validation is minimal, so it is up to the user to ensure that all the elements referred to in the SAMRecords " +
```

Comment:
also, coudl you make use of  replaceCramHeader if it's a cram file? (or open an issue to do this)